python/doc/theory/reliability_sensitivity/taylor_importance_factors.rst

```diff
@@ -7,73 +7,37 @@ The importance factors derived from a Taylor expansion are
 defined to rank the sensitivity of the inputs to
 the output for central dispersion analysis.
 
-Let us denote by :math:`\uX` the input random vector.
-Assume that the marginals of :math:`\uX` are independent.
-Suppose that :math:`Z = h(\ux)` is a real function of the
-input, i.e. :math:`n_Z = 1`.
-Assume that the order 1 Taylor expansion of the function :math:`h`
-at the point :math:`\ux = \muX` is exact, i.e.:
+We denote by :math:`\uX` the input random vector. We assume that the marginals of :math:`\uX` are independent.
 
-.. math::
-    h(\ux) = h(\muX)
-    + \sum_{i=1}^{n_X} \frac{\partial h}{\partial x_i} (\muX) (x_i - \mu_i)
-
-where:
+We note :math:`Y = h(\ux)` with :math:`h: \Rset^d \rightarrow \Rset` a scalar function.
 
--  :math:`\muX` is the mean of the input random vector,
+Refer to :ref:`taylor_expansion_moments` to get details on the expressions of the approximations.
```

Comment:
```suggestion
Please read :ref:`taylor_expansion_moments` for details on the expressions of the approximations.
```

Reply:
I let Refer to be coherent with the other files. I changed the end of the sentence.

Reply:
This is ok for me.